Gopkg.toml

```diff
@@ -83,3 +87,7 @@ required = [
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/e2etemplates"
```

Comment:
Note that this is now imported in here as well (as it is in the project that imports this library).